main.go

```diff
@@ -234,3 +272,30 @@ func resolveTemplate(templateValue string, event *corev2.Event) (string, error)
 
 	return resolved.String(), nil
 }
+
+// https://gist.github.com/homme/22b457eb054a07e7b2fb
+// https://gist.github.com/andelf/5118732
+
+// MIT license (c) andelf 2013
+
+func LoginAuth(username, password string) smtp.Auth {
+	return &loginAuth{username, password}
+}
+
+func (a *loginAuth) Start(server *smtp.ServerInfo) (string, []byte, error) {
+	return "LOGIN", []byte(a.username), nil
+}
+
+func (a *loginAuth) Next(fromServer []byte, more bool) ([]byte, error) {
+	if more {
+		switch string(fromServer) {
+		case "Username:":
+			return []byte(a.username), nil
+		case "Password:":
+			return []byte(a.password), nil
+		default:
+			return nil, errors.New("Unkown fromServer")
```

Comment:
I would definitely suggest fixing the spelling of `Unknown` here. Is there a better error message we can emit that doesn't use an internal variable name `fromServer`?